rust/datafusion/README.md

```diff
@@ -34,59 +34,64 @@ fn main() {
-        Field::new("city", DataType::Utf8, false),
-        Field::new("lat", DataType::Float64, false),
-        Field::new("lng", DataType::Float64, false),
+        Field::new("c1", DataType::Utf8, false),
+        Field::new("c2", DataType::UInt32, false),
+        Field::new("c3", DataType::Int8, false),
+        Field::new("c4", DataType::Int16, false),
+        Field::new("c5", DataType::Int32, false),
+        Field::new("c6", DataType::Int64, false),
+        Field::new("c7", DataType::UInt8, false),
+        Field::new("c8", DataType::UInt16, false),
+        Field::new("c9", DataType::UInt32, false),
+        Field::new("c10", DataType::UInt64, false),
+        Field::new("c11", DataType::Float32, false),
+        Field::new("c12", DataType::Float64, false),
+        Field::new("c13", DataType::Utf8, false),
     ]));
 
     // register csv file with the execution context
-    let csv_datasource = CsvDataSource::new("../../testing/data/csv/uk_cities.csv", schema.clone(), 1024);
-    ctx.register_datasource("cities", Rc::new(RefCell::new(csv_datasource)));
-
-    // simple projection and selection
-    let sql = "SELECT city, lat, lng FROM cities WHERE lat > 51.0 AND lat < 53";
+    let csv_datasource = CsvDataSource::new(
+        "../../testing/data/csv/aggregate_test_100.csv",
+        schema.clone(),
+        1024,
+    );
+    ctx.register_datasource("aggregate_test_100", Rc::new(RefCell::new(csv_datasource)));
 
     // execute the query
+    let sql = "SELECT c1, MIN(c12), MAX(c12) FROM aggregate_test_100 WHERE c11 > 0.1 AND c11 < 0.9 GROUP BY c1";
     let relation = ctx.sql(&sql).unwrap();
-
-    // display the relation
     let mut results = relation.borrow_mut();
 
+    // iterate over result batches
     while let Some(batch) = results.next().unwrap() {
-
         println!(
             "RecordBatch has {} rows and {} columns",
             batch.num_rows(),
             batch.num_columns()
         );
 
-        let city = batch
+        let c1 = batch
             .column(0)
             .as_any()
             .downcast_ref::<BinaryArray>()
             .unwrap();
 
-        let lat = batch
+        let min = batch
             .column(1)
             .as_any()
             .downcast_ref::<Float64Array>()
             .unwrap();
 
-        let lng = batch
+        let max = batch
             .column(2)
             .as_any()
             .downcast_ref::<Float64Array>()
             .unwrap();
 
         for i in 0..batch.num_rows() {
-            let city_name: String = String::from_utf8(city.get_value(i).to_vec()).unwrap();
-
-            println!(
-                "City: {}, Latitude: {}, Longitude: {}",
-                city_name,
-                lat.value(i),
-                lng.value(i),
-            );
+            let c1_value: String = String::from_utf8(c1.value(i).to_vec()).unwrap();
+
+            println!("{}, Min: {}, Max: {}", c1_value, min.value(i), max.value(i),);
```

Comment:
Would it be better to direct the user to an example that's run in CI (to prevent bitrot)?